index.html

```diff
@@ -70,6 +73,26 @@
           </a>
         </div>
       </header>
+      <div class="about-content" x-show="page === '#about'" x-cloak>
+        <p>Wordling is a daily world puzzle geared at young spellers.</p>
+
+        <h1>Features</h1>
+        <ul>
+          <li>Age appropriate words</li>
+          <li>Enter letters in any order</li>
+          <li>3 difficulty settings</li>
+          <li>Emoji colectables</li>
+          <li>Kid-friendly font</li>
+        </ul>
+
+        <h1>How To Play</h1>
+
+        <h1>Creators</h1>
```

Comment:
maybe we should just link to the repo? I think it's nice to highlight it's open source and people could contribute, if they want